src/games/types.ts

```diff
@@ -0,0 +1,10 @@
+export const GameApiRoutes = {
+  wows: {
+    maps: 'wows.encyclopedia.maps',
+    entities: 'wows.encyclopedia.ships'
+  },
+  wot: {
```

Comment:
Wot will be using standard maps.gametactic.eu

Reply:
We haven't developed the WOT stuff yet, everything is a placeholder